pkg/backend/backend.go

```diff
@@ -354,3 +358,38 @@ func (c *backendClient) GetStackResourceOutputs(
 	}
 	return pm, nil
 }
+
+// CreateStackOptions provides options for stack creation. At present,
+// options only apply to the Service.
+type CreateStackOptions interface {
+	Teams() ([]string, error)
```

Comment:
When would this ever error?

Reply:
I removed the error from the parameter, but I'll explain my thinking:

It *would have* errored if we had another implementation of `CreateStackOptions`: e.g. options for handling filestate project versioning, for instance -- where `teams` isn't supported but there are other options that are. The intent would be to keep the signature forward-compatible for other kinds of options. This only works, trivially, because non-service backends can pass `nil` to satisfy the interface.

Moot point, because I changed the signature to remove the error.